types/mailgun-js/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for mailgun-js 0.16
+// Type definitions for mailgun-js 0.17
```

Comment:
Not sure about when the minor version is typically bumped. But this shouldn't be a problem